compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__bit_arrays__discard_sized.snap

```diff
@@ -11,10 +11,10 @@ fn go(x) {
 
 
 ----- COMPILED JAVASCRIPT
-import { makeError } from "../gleam.mjs";
+import { makeError, bitArraySliceToInt } from "../gleam.mjs";
```

Comment:
How come this gets imported? I can't see it used here

Reply:
Fixed. It was happening because of the discarded integer segment.